Packs/AnsibleTower/Integrations/AnsibleTower/AnsibleTower.yml

```
@@ -0,0 +1,423 @@
+category: IT Services
+commonfields:
+  id: AnsibleTower
+  version: -1
+configuration:
+- additionalinfo: Tower server URL
+  display: Ansible URL
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Username
+  hidden: false
+  name: username
+  required: true
+  type: 0
+- hidden: false
+  name: password
+  required: true
+  type: 4
+description: n Scale IT automation, manage complex deployments and speed productivity
+display: AnsibleTower
+name: AnsibleTower
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: If given an inventory id, will retrieve the specific inventory
+      isArray: false
+      name: id
+      required: false
+      secret: false
+    - default: true
+      defaultValue: '1'
+      description: Number of page to retrieve. Default page number is 1
+      isArray: false
+      name: page
```

let's use more descriptive arg name.
```suggestion
      name: page_number
```